device.js

```diff
@@ -34,6 +35,14 @@ var Device = module.exports = function Device() {
 
 Device.DeviceConfig = DeviceConfig;
 
+Device.ActionError = function(statusCode, properties) {
```

Comment:
I don't think the name is working probably. I think we need to set `this.name = 'ActionError';` See test code:

```js
var util = require('util');
var ActionError = function(statusCode, properties) {
  Error.captureStackTrace(this, this.constructor);
  this.name = this.constructor.name;
  this.statusCode = statusCode;
  this.properties = properties;
};
util.inherits(ActionError, Error);

var e = new ActionError(400, {});
console.log(e)
throw e;

{ [] name: '', statusCode: 400, properties: {} }

/Users/ApigeeCorporation/Software/apilife-ui/test.js:12
throw e;
^

    at Object.<anonymous> (/Users/ApigeeCorporation/Software/apilife-ui/test.js:10:9)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:134:18)
    at node.js:961:3
```